atd-vze/src/views/Locations/LocationMap.js

```diff
@@ -81,13 +81,15 @@ export default class LocationMap extends Component {
     };
   }
 
-  _updateViewport = (viewport) => {
+  _updateViewport = viewport => {
     this.setState({ viewport });
   };
 
-  getLatestAerialTimestamp = (timestampArray) => timestampArray.slice(-1)[0];
+  getLatestAerialTimestamp = timestampArray => timestampArray.slice(-1)[0];
 
-  convertNearMapTimeFormat = (date) => moment(date).format("MM/DD/YYYY");
+  convertNearMapTimeFormat = date => {
+    format(parseISO(date), "MM/dd/yyyy");
+  };
```

Comment:
I'm assuming here that `date` is a string, hence the `parseISO` function for parsing strings, but if it's not then this won't work. I couldn't confirm the data type despite all my best console.log efforts because the code was unreachable? I'm not sure how to even get the `convertNearMapTimeFormat` function to run. Does anyone have any insight on what data type `date` is?

Reply:
I took myself down a little rabbit hole here because I couldn't remember how this timestamp is retrieved. The format that comes back here is `/Date(1416873600000)/` which moment.js really surprisingly handles well 🤷, but it is a milliseconds timestamp wrapped with some extra stuff (see the link below).

For `date-fns`, TIL that you can escape characters in the source time string using single quotes when using the [parse](https://date-fns.org/v2.29.3/docs/parse) method. So, I think this works:
```
format(parse(date, "'/Date('T')/'", new Date()), "MM/dd/yyyy");
```

Could you add this link (https://docs.nearmap.com/display/ND/Nearmap+TMS+Integration#NearmapTMSIntegration-Attributes) as a comment somewhere in this code? I should have done that when this code was first written, but I think that I was too excited to get the timestamp for the aerial imagery from the API at all. 😄

Reply:
Also forgot to mention that I tested this locally by replacing the API call with using the helper on a sample return (like `/Date(1416873600000)/`) and then setting that value to state